azext_iot/_params.py

```diff
@@ -819,6 +819,19 @@ def load_arguments(self, _):
             type=int,
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
+        context.argument(
+            "custom_metric_queries",
+            nargs="+",
+            options_list=["--custom-metric-queries", "--cmq"],
+            help="An alternative way of input style(narg key-value pair) to --metrics and and intended to replace "
+            "it in the future.",
+        )
+        context.argument(
+            "custom_labels",
+            nargs="+",
+            options_list=["--custom-labels", "--cl"],
+            help="An alternative way of input style(narg key-value pair) to --labels and intended to replace it in the future.",
```

Comment:
nit: space between style and (

same comment for custom_metric_queries

Reply:
thanks updated!